eclass/vala.eclass

```diff
@@ -103,14 +105,17 @@ vala_depend() {
 # VALA_MAX_API_VERSION, VALA_MIN_API_VERSION, and VALA_USE_DEPEND.
 vala_best_api_version() {
 	local u v
+	local hv_opt="-b"
+	[[ ${EAPI} == 6 ]] && hv_opt=""
```

Comment:
How about making it an array, and avoiding word splitting (or whatever the relevant magic is called)?

```suggestion
	local hv_opt=( -b )
	[[ ${EAPI} == 6 ]] && hv_opt=()
```

Reply:
Seems rather over-engineered to me, given that this will be dropped once EAPI 6 is gone.